src/OrbitGl/FrameTrack.cpp

```diff
@@ -67,7 +67,7 @@ FrameTrack::FrameTrack(CaptureViewElement* parent,
   // TODO(b/169554463): Support manual instrumentation.
 
   // Frame tracks are collapsed by default.
-  collapse_toggle_->SetCollapsed(true);
+  header_->GetCollapseToggle()->SetCollapsed(true);
```

Comment:
Hmm, maybe consider adding a `SetCollapsed` method to `Track` so that we can adjust it locally, in case of changes?